cob.py

```diff
@@ -251,7 +251,7 @@ def _modify_request_before_signing(self, request):
 
 
 def _check_s3_urls(urls):
-    pattern = "s3.*\.amazonaws\.com"
+    pattern = "s3.*\.amazonaws\.com(\.cn)?"
```

Comment:
S3 endpoint for bj region - s3.cn-north-1.amazonaws.com.cn, seems no need to change.